### ISSUE TITLE
Metajuku appeared with no walls and only screens

### ISSUE THREAD

**Issue by punnymarty** [decentraland/unity-renderer#2636]
### What happened?

Walls and skins did not load and everything was transparent. Could see only screens in Metajuku

### Steps to reproduce

Use the Decentraland Beta for Windows and go to 94,21

### Operating System

Windows 10  (10.0.22000) 64bit

### Wallet

Metamask

### CPU

Intel(R) Core(TM) i3-6100U CPU @ 2.30GHz

### GPU

Intel(R) HD Graphics 520 6063MB

### RAM

12126MB

### Kernel Version

1.0.0-2141827118.commit-450c363

### Unity Version

desktop-main.commit-64822b7

### Realm

dg

### NameTag

Murty#7391

**Comment by punnymarty** [comment]
> Hi @punnymarty are you the developer of the scene? We don't know exactly what is expected to be shown or its behavior

No, I am definitely not the developer. I am just expected to see what I would see if I access Metajuku from a browser. But from the windows client, I only see video screens everywhere, but no walls and skins or any other features. Its just flying screens and invisible walls everywhere

**Comment by dalkia** [comment]
Could not reproduce. Currently seeing the same thing on web as in desktop

https://images.zenhubusercontent.com/337227404/5012fcfb-53b5-4b8c-81c4-392d59d2a47f/metajuku_web_1.mp4

https://images.zenhubusercontent.com/337227404/720ae939-ca0c-4010-a6dc-148eeeacb151/metajuku_dekstop_1.mp4

**Comment by punnymarty** [comment]
Hey,
Thank you very much for taking the time to test this out and giving me the confirmation.
Then it should be hardware limitations on my laptop that is causing the issue.

Best Regards,Murtuza

    On Wednesday, 24 August 2022 at 09:59:13 pm GMT+2, Juan Ignacio Molteni ***@***.***> wrote:  
 
 


Could not reproduce. Currently seeing the same thing on web as in desktop

https://images.zenhubusercontent.com/337227404/5012fcfb-53b5-4b8c-81c4-392d59d2a47f/metajuku_web_1.mp4

https://images.zenhubusercontent.com/337227404/720ae939-ca0c-4010-a6dc-148eeeacb151/metajuku_dekstop_1.mp4

**Comment by dalkia** [comment]
@kwspgoliasz @kwssbocian Could you confirm if this error happen in your hardware?

1. Go to scene 94,21 (Matajuku) in web and in desktop
2. Check that the same building is visible

**Comment by kwspgoliasz** [comment]
@dalkia We've checked the location on the Web Client, Windows Client and Mac client, however, the issue could not be reproduced. The location and items loaded correctly for both me and @kwssbocian .

**Comment by punnymarty** [comment]
Thanks, I checked again and my Decentraland desktop app updated. Post-update the above issue has been resolved. Everything however still takes an eternity and high resources to load, which makes it for people with limited system capabilities a difficulty.

**Comment by dalkia** [comment]
Thank you so much @punnymarty for your active involvement in this bug. Is good to know that the issue has been resolved with the latest update.

We'll keep your comment in mind, and we are constantly working to optimize DCL for it to run smoothly on as much hardware as possible.
